staging/src/k8s.io/apiserver/pkg/server/healthz.go

```diff
@@ -101,8 +101,8 @@ func (s *GenericAPIServer) installReadyz() {
 	s.readyzLock.Lock()
 	defer s.readyzLock.Unlock()
 	s.readyzChecksInstalled = true
-	healthz.InstallReadyzHandlerWithHealthyFunc(s.Handler.NonGoRestfulMux, func() {
-		// note: InstallReadyzHandlerWithHealthyFunc guarantees that this is called only once
+	healthz.InstallReadyzHandlerWithCustomFunc(s.Handler.NonGoRestfulMux, func() {
+		// note: InstallReadyzHandlerWithCustomFunc guarantees that this is called only once
```

Comment:
undo the rename?

Reply:
I could but now it is more like `WithHealthyFuncAndDisabledStacktrace`  but OK.

Reply:
done.